improver/calibration/ensemble_calibration.py

```diff
@@ -410,7 +470,7 @@ def _process_points_together(
     def process(
```

Comment:
Does the returns statement for this function need updating to say about the extra beta coeffs?

Reply:
I've extended this comment.